spk/squidguard/Makefile

```diff
@@ -17,6 +17,12 @@ DISPLAY_NAME = SquidGuard
 HOMEPAGE = http://www.squidguard.org/
 LICENSE  = GPL - Copyright © 1990,2007 Oracle.  All rights reserved for berkeley db library. See LICENSE file.
 
+include ../../mk/spksrc.common.mk
+
```

Comment:
obsolete include.
we don't need to include this (or spksrc.arch.mk) to use arch variables with `UNSUPPORTED_ARCHS` as make resolved those variable lately.
_we need those includes for conditional checks within the Makefile only_

```suggestion
```